backend/apps/blogs/mutations.py

```diff
@@ -0,0 +1,90 @@
+import graphene
+from django.shortcuts import get_object_or_404
+
+
+from .models import BlogPost as BlogPostModel
+from .types import BlogPostType
+from graphql_jwt.decorators import permission_required
+
+
+class CreateBlogPost(graphene.Mutation):
+    class Arguments:
+        title = graphene.String()
+        text = graphene.String()
+        author_id = graphene.ID()
+
+    ok = graphene.Boolean()
+    blog_post = graphene.Field(BlogPostType)
+
+    @permission_required("blogs.add_blogpost")
+    def mutate(
+        self,
+        info,
+        title,
+        text,
+        author_id,
+    ):
+
+        blog_post = BlogPostModel.objects.create(
+            title=title,
+            text=text,
+            author_id=author_id,
+        )
+
+        ok = True
+        return CreateBlogPost(blog_post=blog_post, ok=ok)
```

Comment:
I know this is done elsewhere in the application, but might as well just do
```suggestion
        return CreateBlogPost(blog_post=blog_post, ok=True)
```